content/applications/sales/sales/sales_quotations.rst

```diff
@@ -0,0 +1,138 @@
+:show-content:
+:hide-page-toc:
+
+================
+Sales quotations
+================
+
+A *sales quotation* or quote is a document sent to a customer that outlines the estimated costs and
+terms for goods and services. Once accepted, a quotation can be converted into a sales order, which
+serves as the final agreement before delivery and invoicing.
+
+Sales flow overview
+===================
+
+Quotations fit into a broader sales flow that connects different stages of customer interactions
+from initial interest to payment.
+
+The typical flow follows these steps:
+
+#. *Quotation* : A proposal sent to the customer with product details and pricing.
+#. *Sales order*: Created automatically when the customer accepts the quotation, confirming the sale.
+#. *Delivery* (if applicable): Products are shipped or services are delivered.
+#. *Invoice*: The final bill is issued based on the sales order or delivered products/services.
+#. *Payment*: The customer settles the invoice, completing the sales cycle.
+
+This flow helps businesses track the entire lifecycle of a sale while keeping information consistent
+across apps.
+
+In Odoo, quotations are configured in the **Sales** app. They can also be generated from other apps
+as part of the sales workflow:
+
+- CRM: :doc:`Convert opportunities <../crm/acquire_leads/send_quotes>` into quotes to follow up on
```

Comment:
This list is a good start. 

We're missing supply-chain and accounting apps, though, which are 
a) arguably the most important in Odoo, generally speaking 
b) and the most relevant in this context since business ops are closely integrated with the varying stages of quotes/SOs.